src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -156,10 +145,11 @@ private Arm(ArmParameters parameters) {
     motor =
         new TalonFXAdapter(logPrefix, talonFX, talonFXConfigs.MotorOutput, RADIANS_PER_ROTATION);
     relativeEncoder = motor.getEncoder();
-    relativeEncoder.setPosition(absoluteEncoder.getAngle());
+    relativeEncoder.setPosition(
+        Math.toRadians(
+            92)); // TODO: Parametize starting position? Different arms start in different degrees.
```

Comment:
Yes, this should be parameterized.
Nit: please don't put long comments at the ends of lines of code, it causes the formatter to split the code into 3+ lines.
```suggestion
   // TODO: Parametize starting position? Different arms have different starting positions.
   relativeEncoder.setPosition(Math.toRadians(92));
```